src/components/Sidebar/index.js

```diff
@@ -1,14 +1,48 @@
 import React, { Component } from 'react';
 import './index.css';
-import { CreatePostForm } from '../CreatePostForm';
+import { BrowserRouter as Router, Link } from 'react-router-dom'
 
 export class Sidebar extends Component {
+  constructor() {
+    super()
```

Comment:
Why did you use constructor () to define state?
If it not necessary please do it like in other components.